lighthouse-core/config/default-config.js

```diff
@@ -383,6 +384,7 @@ const defaultConfig = {
       auditRefs: [
         {id: 'first-contentful-paint', weight: 3, group: 'metrics'},
         {id: 'first-meaningful-paint', weight: 1, group: 'metrics'},
+        {id: 'largest-contentful-paint', weight: 0, group: 'metrics'},
```

Comment:
@paulirish do I recall correctly that we intend to give this one of the highest weights right out of the gate?

Reply:
yup. i'll do that in a followup PR. :)